code-challenges.test.js

```diff
@@ -13,47 +13,147 @@
+    expect(multiply([24, 27, 30, 33, 36])).toEqual([72, 81, 90, 99, 108])
+  })
+})
 
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
+//Create a function named 'multiply' that takes an array as a parameter.
+//Use the .map() method to traverse the array and multiply all values by 3.
+//Place all new values into a new array and return that array.
+const multiply = (array) => (array.map((value) => value*3))
+// console.log(multiply(numbersArray1)) --> [ 18, 21, 24, 27, 30 ]
+// console.log(multiply(numbersArray2)) --> [ 72, 81, 90, 99, 108 ]
+
+//Note: I wasnt sure if i was supposed to have the variables in the test or not so I will do that for now on.
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+//Create a test for a function named 'divisbleBy3'.
+describe('divisibleBy3', () => {
+
+  //Explain how the function is supposed to perform
+  it('takes a object as an argument and decides if the number inside it is evenly divisible by three or not', () => {
+
+    //variables for the expect
+    const object1 = { number: 15 }
+    // Expected output: "15 is divisible by three"
+    const object2 = { number: 0 }
+    // Expected output: "0 is divisible by three"
+    const object3 = { number: -7 }
+    // Expected output: "-7 is not divisible by three"
+
+    //hardcode what you would pass as an argument and what the 
+    //expected outcomes will be.
+    expect(divisibleBy3(object1)).toEqual("15 is divisible by three")
+    expect(divisibleBy3(object2)).toEqual("0 is divisible by three")
+    expect(divisibleBy3(object3)).toEqual("-7 is not divisible by three")
+  })
+})
 
 // b) Create the function that makes the test pass.
+//create a function named 'divisibleBy3' that takes an object as an argument.
+const divisibleBy3 = (object) => {
+  //If the number found within the object is divisible by 3 return, use string
+  //interpolation to state that it is divisible by 3.
+  if (object.number % 3 === 0){
+    return `${object.number} is divisible by three`
+
+    //If the number found within the object is not divisible by 3 return, use string
+    //interpolation to state that it is not divisible by 3.
+  } else if (object.number % 3 !== 0){
+    return `${object.number} is not divisible by three`
+
+    //If all ifs fails, something went wrong
+  } else {
+    return 'Something wwent wrong'
+  }
```

Comment:
Great job of using your else statement as a catch all and thinking about those different scenarios!